loopin.go

```diff
@@ -406,20 +406,19 @@ func (s *loopInSwap) waitForHtlcConf(globalCtx context.Context) (
 		return nil, err
 	}
 
-	for {
+	var conf *chainntnfs.TxConfirmation
```

Comment:
nit: Imho the original version was just as clean as this one if not a bit cleaner. Of course it's subjective.

Reply:
I need this for the change in the next commit